app/uk/gov/hmrc/customs/declaration/connectors/filetransmission/FileTransmissionConnector.scala

```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
```

Comment:
- A question - did you think about pulling the upscan etc data from `TestData.scala` into a separate file/files? It would then leave `TestData.scala` with "core" data only and reduce its size.

Reply:
As discussed, does not need to be done.